ext/session/tests/session_create_id_basic.phpt

```diff
@@ -16,6 +16,10 @@ echo "*** Testing session_create_id() : basic functionality ***\n";
 
 // No session
 var_dump(session_create_id());
+var_dump(session_create_id(''));
```

Comment:
An empty string should not be valid.

Reply:
So the first param of this function is the prefix, which has a default value of an empty string.